app/models/reward.rb

```diff
@@ -0,0 +1,4 @@
+class Reward < ActiveRecord::Base
+  validates :name, presence: true, uniqueness: true
```

Comment:
nice! i forgot to do this using active record and put in some bulky ruby logic to do this--need to fix that on mine